### PR TITLE
got the functionality of the artistName and songTitle being searched.…

### PR DESCRIPTION
**Pull request by CaseyVanAlstyne**
… Also added artist image and link for bio.

### CODE REVIEW

File: script.js [hunk 1]
```diff
@@ -44,10 +44,13 @@ $(document).ready(function () {
...
-                console.log("this is 100", result[0].artworkUrl100)
-                console.log("this is artist view url", result[0].artistViewUrl)
-                // $("#imageOutput").prepend(artistViewUrl);
+                // console.log("current log", result)
+                // console.log("this is 100", result[0].artworkUrl100)
+                // console.log("this is artist view url", result[0].artistViewUrl)
+                $("#artistInfo").attr("href", result[0].artistViewUrl);
+                $("#imageOutput").attr("src", result[0].artworkUrl100);
+                $("#imageOutput").attr("alt", result[0].artistName);
+                $("#imageOutput").attr("alt", result[0].artistName);
```
Comment (ghost) [inline script.js:53]: Hey Casey did you mean to have lines 52 and 53 to be the same?

**Review by ghost — DISMISSED** [review]
Hey Casey iTunes function looks great! For lines 52 and 53, did you mean to have it be the same lines of code?

File: script.js (hunk 1 above)
Comment (CaseyVanAlstyne) [reply script.js:53]: Did not mean to have the same line twice.

**Review by ghost — APPROVED** [review]
Looks great! Second API works!!!

**Pull request merged**